### DISCUSSION TITLE
Question About Gateway4+

### DISCUSSION THREAD

**Discussion by CrushDope** [apache/skywalking#10769] [Q&A]
Hello developer, recently I used SpringCloud2022.0.2 version for project development, gateway module used Gateway4.0.4 version, skywalking used version 9.4.0, agent used version 8.15.0, but I did not find it in skywalking topology Gateway module, I searched the plug-in directory in the optional-plugins in the skywalking-agent project and found no Gateway4-related plug-ins. Will the gateway4 version be compatible in the future?

**Comment by wu-sheng** [comment]
SkyWalking is built by global contributors. If someone contributed to this, we will have.
There is no simple answer about `Will the gateway4 version be compatible in the future?`

**Reply by CrushDope** [reply]
Thank you very much for your reply. I would like to know how to use it with skywalking in the Gateway4 module, because I did not find my Gateway module in the topology in skywalking, and my requests are all forwarded from the gateway.

**Reply by wu-sheng** [reply]
I can't see that is on the supported list. Please follow the doc.
AFAIK, there is no one declared that is tested.

**Reply by CrushDope** [reply]
Well, great to hear from you
